app/models/core_data_connector/title_source.rb

```diff
@@ -0,0 +1,7 @@
+module CoreDataConnector
```

Comment:
Interesting. Do we anticipate titles being reused? Why not related the `title` directly to the `work` or `instance`?

Reply:
Should titles just be another model that users can add to their project?

Reply:
I think the titles for a source would be analogous to how we're allowing multiple names for a place/person/organization.